src/react/AuthenticatedPageRoute.test.jsx

```diff
@@ -45,6 +47,7 @@ describe('AuthenticatedPageRoute', () => {
       </AppContext.Provider>
     );
     history.push('/authenticated');
+    global.location.href = 'http://localhost/authenticated';
```

Comment:
nit: I'm curious why it's necessary to update `global.location.href` now when it wasn't needed before. Is it just because we're now doing `delete global.location;` rather than `delete global.location.assign;`, which wouldn't affect `location.href`?

Reply:
Yeppers!